ale_linters/eruby/erubylint.vim

```diff
@@ -0,0 +1,27 @@
+" Author: Matthias Guenther - https://wikimatze.de
+" Description: erb-lint for eruby/erb files
+
+function! ale_linters#eruby#erubylint#Handle(buffer, lines) abort
```

Comment:
Got, if I'm with https://github.com/w0rp/ale/pull/497/commits/a94e6c7f282e8b6951263f409f73e87001232175 on the right way, I'll try to add tests.